imports/collections/schemas/address.js

```diff
@@ -17,6 +17,8 @@ const withoutCodeCountries = ["AO", "AG", "AW", "BS", "BZ", "BJ", "BW",
  * @type {SimpleSchema}
  * @property {String} _id
  * @property {String} fullName required
+ * @property {String} fistName
```

Comment:
@kieckhafer Just pointing out this misspelling in case you want to fix in the next PR you do.

Reply:
Thanks, fixed it in https://github.com/reactioncommerce/reaction/pull/5071